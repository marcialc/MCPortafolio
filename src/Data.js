let Data = {
  name: "Marcial Cabrera",
  headline: "Software Engineer / Front-end Developer / Mobile Developer",
  socialLinks: [
    {
      name: "linkedin",
      url: "https://www.linkedin.com/in/marcialc",
      className: "fa fa-linkedin"
    },
    {
      name: "github",
      url: "http://github.com/marcialc",
      className: "fa fa-github"
    },
  ],
  aboutme:
    "Currently a Senior in Florida International University (FIU) Pursing a Bachelors of Science in Computer Science. Very passionated about about my field and whats to come and the new tehcnologies and challenges it brings.",
  address: "Miami, FL 33178",
  email: "marcialandres06@gmail.com",
  education: [
    {
      UniversityName: "Florida International University (FIU)",
      major: "B.S Computer Science",
      endDate: "Fall 2019"
    }
  ],
  work: [
    {
      CompanyName: "American Express",
      title: "Mobile Engineer Intern",
      description:
        "Conducted code reviews and tested ongoin sprints while being a core memeber of a Agile team. Designed and developed on existing movile application on multiple channels such as React Native, Android and Java services/APIs.",
      startDate: "June 2019",
      endDate: "August 2019"
    },
    {
      CompanyName: "CodeArt",
      title: "Programming Teacher",
      description:
        "As an instructor at CodeArt, my goals are to help the cause that CodeArt is continuing to support by opening the doors to the previously male-dominated field of coding, CodeArt inspires young women to explore coding, showcasing their talents, and pursue careers in this growing industry.",
      startDate: "June 2019",
      endDate: "Present"
    }
  ],
  programmminglanguages: [
    {
      skillname: "JAVA"
    },
    {
      skillname: "JavaScript"
    },
    {
      skillname: "Swift"
    },
    {
      skillname: "TypeScript"
    },
    {
      skillname: "Kotlin"
    },
  ],
  projects: [
    {
      title: "Full-Stack",
      name: "Portfolio",
      description: "Developed front-end using Redux and React.js, also implemented back-end using node.js, java spring and MongoDB. Jest testing framework was used for code consistency and unit testing.",
      date: "09 / 2019"
    },
    {
      title: "Neural Network / iOS App",
      name: "ParasiteAI",
      description: "For creating the neural network we used Flaks web app and it was hosted in Google Could App Engine. Created an iOS app that interacted with the neural network using 8base. It also won “best design” and third place overall.",
      date: "02 / 2019"
    },
    {
      title: "React-Native App",
      name: "Bank-Health",
      description: "Build a react native app, that told you about your bank health based on your transactions and account balance using fake data from City Bank API. Stored and saved all data in firebase. Used express framework from node.js for the back-end. It was also connected to google assistance for a hand-free experience.",
      date: "09 / 2018"
    },
  ],
};

export default Data;
