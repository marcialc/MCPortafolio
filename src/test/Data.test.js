import renderer from 'react-test-renderer';
import Data from '../Data';

test('My info', () => {
    const name  = renderer.create(Data.name);
    let tree = name.toJSON();
    expect(tree).toMatchSnapshot();

    const headline  = renderer.create(Data.headline);
    tree = headline.toJSON();
    expect(tree).toMatchSnapshot();

    const aboutme  = renderer.create(Data.aboutme);
    tree = aboutme.toJSON();
    expect(tree).toMatchSnapshot();
});