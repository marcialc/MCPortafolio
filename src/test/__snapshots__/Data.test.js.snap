// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My info 1`] = `"Marcial Cabrera"`;

exports[`My info 2`] = `"Software Engineer / Front-end Developer / Mobile Developer"`;

exports[`My info 3`] = `"Currently a Senior in Florida International University (FIU) Pursing a Bachelors of Science in Computer Science field. Very passionated about about my field and whats to come and the new tehcnologies and challenges it brings."`;
